src/data/polynomial/degree/definitions.lean

```diff
@@ -1156,6 +1156,20 @@ else if hq0 : q = 0 then  by simp only [hq0, degree_zero, mul_zero, with_bot.add
 else degree_mul' $ mul_ne_zero (mt leading_coeff_eq_zero.1 hp0)
     (mt leading_coeff_eq_zero.1 hq0)
 
+lemma degree_le_of_dvd : (q ≠ 0) → (p ∣ q) → degree p ≤ degree q :=
```

Comment:
If the first tactic would otherwise be `intro`, mathlib custom is to put the arguments "before the colon":
```suggestion
lemma degree_le_of_dvd (hqnz : q ≠ 0) (hpq : p ∣ q), degree p ≤ degree q :=
```
Then you can delete `intros hpnz hpq`.